src/spicy/manager.cc

```diff
@@ -897,14 +897,21 @@ void Manager::disableReplacedAnalyzers() {
         if ( file_mgr->Lookup(replaces, false) || packet_mgr->Lookup(replaces, false) )
             reporter->FatalError("cannot replace '%s' analyzer with a protocol analyzer", replaces);
 
-        auto tag = analyzer_mgr->GetAnalyzerTag(replaces);
-        if ( ! tag ) {
+        auto component = analyzer_mgr->Lookup(replaces, false);
+        if ( ! component ) {
             SPICY_DEBUG(hilti::rt::fmt("%s is supposed to replace protocol analyzer %s, but that does not exist",
                                        info.name_analyzer, replaces));
 
             continue;
         }
 
+        auto tag = component->Tag();
+        if ( analyzer_mgr->HasComponentMapping(tag) )
+            reporter->FatalError(
+                "%s: protocol analyzer %s is already mapped to a different analyzer; cannot replace an analyzer "
```

Comment:
Nit: It's a bit unfortunate that we currently cannot tell the user which previous analyzer also made a replacement since there is no API for us to query that. If we had e.g., 

```cpp
// Returns component mapped over this one if there is a mapping defined, else `nullptr`.
`Component* GetComponentMapping(const zeek::Tag& tag) const`
```

we could do that (and replace all uses of `HasComponentMapping` as well).

Reply:
Yeah, I thought about that but then figured it's fine, this should be rare enough in the first place